5EUNG/app.js

```diff
@@ -56,11 +56,144 @@ app.post('/signup', async (req, res) => {
   res.status(201).json({ message: "userCreated" });
 });
 
+app.post('/posts', async (req, res) => {
+  const { title, content, postImage, userId } = req.body
+
+  await appDataSource.query(
+    `INSERT INTO posts(
+      title,
+      content,
+      post_image,
+      user_id
+      ) VALUES (?, ?, ?, ?);
+    `,
+    [title, content, postImage, userId]
+  )
+
+  res.status(201).json({ message: "postCreated" });
+})
+
+app.get('/posts', async (req, res) => {
+  await appDataSource.query(
+    `SELECT
+      u.id AS userId,
+      u.profile_image AS userProfileImage,
+      p.id AS postingId,
+      p.post_image AS postingImageUrl,
+      p.content AS postingContent
+   FROM users u
+   INNER JOIN posts p ON u.id = p.user_id`,
+    (err, rows) => {
+      res.status(200).json({ data: rows });
+    }
+  );
+});
+
+app.get('/user_post/userId/:userId', async (req, res) => {
```

Comment:
해당 API 는 유저의 게시글을 확인하는 것으로 짐작됩니다. 이미 메소드 이름에 충분히 의미가 부여되었으니 URI는 요청에 맞는 자원을 기준으로 작성해 주세요.
```suggestion
app.get('/post/user/:userId', async (req, res) => {
```